index.js

```diff
@@ -26,13 +26,23 @@ module.exports = function(options, pattern) {
 						.filter(pattern.test.bind(pattern))
 						.forEach(function(name) {
 							pending++;
-							new bless.Parser({ output: name, options : options })
+							basename = name.substring(name.lastIndexOf('/') + 1);
+							if (basename.lastIndexOf('.') !== -1) {
+								basename = basename.substring(0, basename.lastIndexOf('.'));
+							}
```

Comment:
Could you use [`path.parse`](https://nodejs.org/api/path.html#path_path_parse_pathstring) instead? Not all operating systems (i.e. Windows) use `/` as a path separator, using the NodeJS provided modules here gives cross platform support for free.